components/infobox/wikis/valorant/infobox_person_player_custom.lua

```diff
@@ -136,18 +143,30 @@ function CustomPlayer._createRole(key, role)
 	end
 end
 
+function CustomPlayer._isNotPlayer(role)
+	local roleData = _ROLES[(role or ''):lower()]
+	return roleData and (roleData.talent or roleData.staff)
+end
+
 function CustomPlayer:defineCustomPageVariables(args)
 	-- isplayer needed for SMW
-	local roleData
-	if String.isNotEmpty(args.role) then
-		roleData = _ROLES[args.role:lower()]
-	end
-	-- If the role is missing, assume it is a player
-	if roleData and roleData.isplayer == false then
+	if CustomPlayer._isNotPlayer(args.role) then
 		Variables.varDefine('isplayer', 'false')
 	else
 		Variables.varDefine('isplayer', 'true')
 	end
 end
 
+function CustomPlayer:getPersonType(args)
+	local roleData = _ROLES[(args.role or ''):lower()]
+	if roleData then
+		if roleData.staff then
+			return { store = 'Staff', category = 'Staff' }
```

Comment:
```suggestion
			return {store = 'Staff', category = 'Staff'}
```
keep it consistent with the rest of the module

same applies for lines 166 and 169 too